server/src/test/java/keywhiz/service/permission/PermissionCheckTest.java

```diff
@@ -0,0 +1,51 @@
+package keywhiz.service.permission;
+
+import com.codahale.metrics.MetricRegistry;
+import com.google.inject.Provider;
+import keywhiz.service.permissions.Action;
+import keywhiz.service.permissions.AlwaysAllowPermissionCheck;
+import keywhiz.service.permissions.AlwaysFailPermissionCheck;
+import keywhiz.service.permissions.KeywhizPrincipal;
+import keywhiz.service.permissions.PermissionCheck;
+import keywhiz.service.permissions.PermissionCheckImpl;
+import keywhiz.service.permissions.PermissionCheckProvider;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PermissionCheckTest {
+
+  @Rule public MockitoRule mockito = MockitoJUnit.rule();
+
+  @Mock MetricRegistry metricRegistry;
+  @Mock KeywhizPrincipal principal;
+  @Mock Object target;
+  Provider<PermissionCheck> permissionCheckProvider;
+  PermissionCheckImpl permissionCheck;
+  String add = Action.ADD;
+  String create = Action.CREATE;
+  String delete = Action.DELETE;
```

Comment:
I would just use the constants directly in the code instead of storing them in locals, that way it's obvious that they're constants when you see them in use.